lib/YuiRestClient/Wait.pm

```diff
@@ -3,12 +3,13 @@
 package YuiRestClient::Wait;
 use strict;
 use warnings;
-
+use testapi;
 use DateTime;
 
 sub wait_until {
     my (%args) = @_;
     $args{timeout} //= 10;
+    $args{timeout} *= get_var('TIMEOUT_SCALE', 1);
```

Comment:
remember that we don't use "in general" the testapi in the framework, other than in the Test Layer, so you need to introduce a new parameter and pass it from test layer to this encapsulate component which shouldn't know anything about `get_var` function or other testapi dependencies, so it is decoupled.